pkg/smartcontract/manifest/standard/nep30.go

```diff
@@ -0,0 +1,23 @@
+package standard
+
+import (
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract"
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract/manifest"
+)
+
+// Nep30 is a NEP-30 Standard describing smart contract verify method functionality.
+var Nep30 = &Standard{
+	Manifest: manifest.Manifest{
+		ABI: manifest.ABI{
+			Methods: []manifest.Method{
+				{
+					Name:       "verify",
+					Safe:       true,
+					ReturnType: smartcontract.BoolType,
+				},
+				// nil parameters implies no parameters check, NEP-30 allows an
+				// undefined number of parameters for verify.
```

Comment:
https://github.com/nspcc-dev/neo-go/pull/3978#discussion_r2293384219 not resolved.

This comment is a part of method declaration, it should be placed to the method declaration before `},`.